index.html

```diff
@@ -228,45 +226,77 @@
-with a time value equal to the time immediately before the user agent starts
-the <a href="http://www.w3.org/TR/html5/browsers.html#unloading-documents">unload event</a> of the previous document.
+<section id='PerformanceResourceTiming'>
+<h3>Relation with the <a href="http://www.w3.org/TR/resource-timing/#idl-def-PerformanceResourceTiming">PerformanceResourceTiming</a> interface</h3>
 
-If there is no previous document or the previous document has a
-different <a href="http://tools.ietf.org/html/rfc6454">origin</a> than the
-current document, this attribute MUST return a <a href="http://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
-with a time value equal to zero.</p>
-</dd>
+<p>Only the <a>current document</a> resource gets included as the <em>only</em> <a>PerformanceNavigationTiming</a> object in the Performance Timeline of the relevant context.</p>
 
-<dt>readonly attribute DOMHighResTimeStamp unloadEventEnd</dt>
+<p>The <a>PerformanceNavigationTiming</a> interface extends the following attributes of the <a href="http://www.w3.org/TR/resource-timing/#idl-def-PerformanceResourceTiming">PerformanceResourceTiming</a> interface:</p>
+
+<dl class='attributes'>
+  <dt id='widl-PerformanceNavigationTiming-initiatorType'><code>initiatorType</code></dt>
+  <dd>This attribute MUST return the empty <code>DOMString</code> "".</dd>
+
+  <dt id='widl-PerformanceNavigationTiming-workerStart'><code>workerStart</code></dt>
+  <dd>If the <a>current document</a> has an <a href="http://www.w3.org/TR/service-workers/#dfn-containing-service-worker-registration">active service worker registration</a> ([[!SERVICE-WORKERS]]), this attribute MUST return the time immediately before the user agent <a href="http://www.w3.org/TR/service-workers/#service-worker-concept">ran the worker</a> required to service the request, or if the worker was already available, the time immediately before the user agent <a href="http://www.w3.org/TR/service-workers/#on-fetch-request-algorithm">fired an event named `fetch`</a> at the <a href="http://www.w3.org/TR/service-workers/#dfn-active-worker">active worker</a>. Otherwise, if there is no active worker this attribute MUST return zero.</dd>
+
+</dl>
+
+</section>
+
+<section id="sec-PerformanceNavigationTiming">
+<h3>The <code>PerformanceNavigationTiming</code> interface</h3>
+
+<!-- from domainLookupEnd -->
+<div class="note">
+<p>Checking and retrieving contents from the
+<a href="https://tools.ietf.org/html/rfc7234">HTTP cache</a> [[RFC7234]]
+is part of the <a href="http://www.w3.org/TR/html5/infrastructure.html#fetch" title='fetch'>
+fetching process</a>. It's covered by the
+<a href="#widl-PerformanceNavigationTiming-requestStart">requestStart</a>,
+<a href="#widl-PerformanceNavigationTiming-responseStart">responseStart</a> and
+<a href="#widl-PerformanceNavigationTiming-responseEnd">responseEnd</a> attributes.</p>
+</div>
+
+<!-- from requestStart -->

```

Comment:
Folks have been asking us many times why we didn't have a requestEnd. Maybe move the note into ResourceTiming?